### DISCUSSION TITLE
Problem with transparent Cube

### DISCUSSION THREAD

**Discussion by Leanxp** [google/model-viewer#3302] [Q&A]
I have 3 models

1. Door frame
2. Astronaut
3. Cube transparent 

I've been using blender to get that transparent cube to be in the first layer and not to show astronaut, but it's impossible...
This is my idea, like a portal AR.

https://imgur.com/Hz1vAaW

**Comment by elalish** [comment]
Yeah, glTF is based on physical materials, and it seems like what you want is a non-physical material, so that's likely the trouble. I think you'd need some kind of new glTF extension that specifies the behavior you want.

**Reply by Leanxp** [reply]
Is there no way to enable a transparent material that makes that layer (material) above the astronaut object?

**Reply by elalish** [reply]
By "above" do you mean "occludes"?

**Reply by Leanxp** [reply]
Exactly!! I need to deploy an "hider" (invisible material) but it's impossible in gltf... 
Something like this: https://imgur.com/OmsTxqu

**Reply by elalish** [reply]
Yes, as I say, this is a non-physical material (there's nothing you could build in the real world that would do this), hence why it's not in glTF. However, it's a good idea, and hence could be a useful extension to add.
